docs/pages/accounts/audit-logs.mdx

```diff
@@ -0,0 +1,61 @@
+
+Audit logs are available for enterprise plan customers only. Immediately after subscribing, some of the logs will become available immediately as they could have been used internally by Expo. Some other types of logs will start to be collected after the subscription is activated.
+
+They can be accessed for any account under [expo.dev/accounts/[account]/settings/audit-logs](https://expo.dev/accounts/[account]/settings/audit-logs).
+
+## Audit log use cases
+
+### Permission monitoring
+
+Audit logs can be used to track user invitations and permission changes within your organization. An example security event could include an employee account being compromised and used to invite an attacker into an organization and change their permission to admin.
+
+In this scenario, audit logs would record which employee account invited an account and modified permissions, and since audit logs are immutable, the attacker would not be able to delete this recorded history. Other organization members would be able to review the audit logs to determine which account was compromised and take action to revoke the attacker's permissions and secure the employee's account.
+
+### Access history
+
+An Expo organization account can include many projects where development access is controlled by distribution certificates assigned to individual teams. When devices are granted to join these teams, it is important to track when access is granted and removed for historical record keeping. While a device may not currently be included in an Apple team, it may be useful to see who previously had access to the team in the event of an internal security incident.
+
+The Apple devices listed within Expo's team settings will only show devices that are currently registered to an account, but with the creation of audit logs, historical modifications of Apple teams and devices can be viewed.
+
+## Where are audit logs in use?
+
+Audit logs have been enabled for several entities that will offer the highest level of action tracking. While we will be adding more entities to audit logs in the future, at the time of this blog post, we have enabled audit logs for the following entities:
+
+- Android app credentials
+- Android keystores
+- Apple devices
+- Apple distribution certificates
+- Apple provisioning profiles
+- Apple teams
+- App store connect api keys
+- Google service account keys
+- iOS app credentials
+- User permissions
+- Apps
+- Accounts
+- User invitations
+
+## Understanding Audit Logs
+
+Audit Log entries include the following fields:
+
+- Actor: the account actor that performed the particular action
+- Entity Type: The object that was modified
+- Mutation Type: The type of modification, CREATE, UPDATE, DELETE
+- Created At: When the particular action was performed
+
+Additionally, the three dots at the end of an Audit Log row can be opened to view any metadata relevant to the Audit Log.
```

Comment:
```suggestion
Additionally, clicking on an Audit log row, you can view the metadata relevant to that log.
```

I think our UI might have changed as I'm not seeing a three-dotted menu icon. However, clicking it to open a log's metadata seems to work.

![CleanShot 2024-10-28 at 20 12 15@2x](https://github.com/user-attachments/assets/7636eef5-b7c5-43b8-9b45-08eb3fece29c)